core/src/components/select/select.ionic.outline.scss

```
@@ -50,3 +50,10 @@
 
   box-sizing: border-box;
 }
+
+// Focus
+// ---------------------------------------------
+
+:host(.ion-focused.select-fill-outline) {
+  --border-color: var(--highlight-color);
```

```suggestion
  --border-color: var(--highlight-color);
  --border-width: #{globals.$ion-border-size-050};
```
When focused, the border width also changes.